cek/cost_model.go

```diff
@@ -75,21 +75,15 @@ func sizeExMem(i int) func() ExMem {
 
 // Return a function so we can have lazy
 // costing of params for the case of constant functions
+//
+//go:inline
 func bigIntExMem(i *big.Int) func() ExMem {
 	return func() ExMem {
-		x := big.NewInt(0)
-
-		if x.Cmp(i) == 0 {
+		if i.Sign() == 0 {
```

Comment:
Please add a comment here noting that the point of calling `.Sign()` is to check for `0` without doing an allocation